backend/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM denoland/deno:alpine
+
+EXPOSE 1039
+
+WORKDIR /app
+
+COPY . .
+
+RUN deno cache mod.ts
+
+CMD ["run", "--allow-net", "mod.ts"]
```

Comment:
Is it the command to run backend? If yes, I suggest we can run it with entrypoint
If we run backend with entrypoint, it won't be overwritten by user

Reply:
What is the difference between `ENTRYPOINT` and `CMD`?

Line 9 is just caching the dependencies and compiling.

Reply:
The commands on `CMD` can be overwritten by user while `ENTRYPOINT` can't. For more details, please see https://www.bmc.com/blogs/docker-cmd-vs-entrypoint/

But for debugging purposes, we can leverage `CMD` first